Project/Assets/ML-Agents/Examples/Startup/Scripts/Startup.cs

```diff
@@ -7,24 +7,48 @@ namespace MLAgentsExamples
     internal class Startup : MonoBehaviour
     {
         const string k_SceneVariableName = "SCENE_NAME";
+        private const string k_SceneCommandLineFlag = "--scene-name";
```

Comment:
I know this is in Examples, but can you prefix this with something like `--mlagents-scene-name`?

Reply:
Sure thing, will change.  I was on the fence about that one in any case but fell on the side of matching the env var.